dockerfiles/mlrun-api/requirements.txt

```diff
@@ -18,3 +18,4 @@ aiosmtplib~=3.0
 pydantic>=1,<2
 # ensure kfp is installed
 mlrun-pipelines-kfp-v1-8[kfp]~=0.3.5; python_version < "3.11"
+grpcio~=1.69.0
```

Comment:
why .69 and not 1.70?

Reply:
As per the lock file, which has been updated in the meantime. Will change to 1.70.0.